lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -28,6 +31,16 @@ def self.get_milestone(repository, release)
         return mile
       end
 
+      # Downloads a file from the given GitHub tag
+      #
+      # @param String repository The repository name (including the organization) [ex: wordpress-mobile/wordpress-ios]
+      # @param String tag The name of the tag we're downloading from [ex: 16.9]
+      # @return [<Sawyer::Resource>] A list of the PRs for the given milestone, sorted by number
+      #
+      def self.get_prs_for_milestone(repository, release)
+        github_client().search_issues("type:pr milestone:16.9 repo:wordpress-mobile/wordpress-ios")[:items].sort_by(&:number)
```

Comment:
You left hardcoded values here both for the repo slug and the tag name to fetch 😅

Reply:
🤦 

Fixed in f4c890bceaa0a8c909dc58303c6ea22604ac24b6